src/java/src/main/java/jcoz/service/JCozServiceImpl.java

```diff
@@ -96,9 +103,11 @@ public int attachToProcess(int localProcessId)
                 }
             }
         } catch (IOException | NumberFormatException
-                | AttachNotSupportedException exception) {
-            exception.printStackTrace();
-            throw new RemoteException("", exception);
+                | AttachNotSupportedException e) {
+            StringWriter stringWriter = new StringWriter();
+            e.printStackTrace(new PrintWriter(stringWriter));
+            logger.error("Got an exception during attachToProcess, stacktrace: {}", stringWriter);
```

Comment:
I'm not sure this error message is needful considering we print the stacktrace in the line above, and then throw the exception right below. If we agree I can remove it in a follow-on commit.